src/main/java/org/jabref/gui/groups/GroupDialogView.java

```diff
@@ -152,6 +154,8 @@ public void initialize() {
                 getDialogPane().lookupButton(ButtonType.OK).setDisable(true);
             }
         });
+
+      //  nameField.requestFocus();
```

Comment:
You can just delete this comment, as you have moved it into the Platform.runLater-block.